WordPress/src/test/java/org/wordpress/android/ui/activitylog/list/filter/ActivityLogTypeFilterViewModelTest.kt

```diff
@@ -36,6 +36,8 @@ class ActivityLogTypeFilterViewModelTest : BaseUnitTest() {
     @Mock private lateinit var parentViewModel: ActivityLogViewModel
     @Mock private lateinit var activityLogStore: ActivityLogStore
 
+    private val remoteSiteId = 0L
```

Comment:
Minor (🔍): Consider making this a constant.

Reply:
Fixed in d1d05d26adf4ccee3b6c4ceb3dc0685e2cc34be0